### ISSUE TITLE
about  "v15" model format

### ISSUE THREAD

**Issue by Nightbringers** [lightvector/KataGo#869]
Added support for a new "v15" model format that adds a nonlinearity to the pass policy head. This change is required for the new larger b28c512nbt neural net that should be ready in the next few months and might become the strongest neural net to use for top-tier GPUs.

hello, i want ask few questions about this new "v15" model format,  what is "adds a nonlinearity to the pass policy head" , is there have some docu about it？ Does it improve a lot?  And does this new "v15" model network architecture have any improvement compared to "b18c384" net？

**Comment by lightvector** [comment]
No, it's a minor change that isn't that important. It possibly makes it slightly easier for the network to fit the correct function to determine when to pass at the very end of the game. It won't have any effect on the rest of the game because for almost all of the game passing is never a good move.
